### DISCUSSION TITLE
Pot | 欧路词典

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#981] [General]
# Pot | 欧路词典

Pot是一款强大的跨平台划词翻译和OCR软件，支持DeepL、OpenAI、谷歌翻译等多个翻译引擎，具备划词翻译、截图翻译、文字识别、语音合成朗读等功能。简洁的界面设计，快捷的操作方式，是您的最后一款翻译软件。支持离线翻译，确保隐私安全，让跨语言交流更加便捷。

https://pot-app.com/docs/api/collection/eudic.html

**Comment by YeHang-GitHub** [comment]
如果使用划词翻译或者截图翻译的话， 这个功能只会把第一次拾取到的词加到生词本而不是翻译界面的词加入生词本，举个例子我拾取pot这个词，如果我不小心拾取到了旁边的词变成了zpot,然后我触发了划词翻译，这个时候我在界面中手动把z给删除了，然后下方的界面显示的就是正确的pot，但是如果我这个时候点击添加到生词本，实际上添加过去的还是之前的zpot而不是pot，哪怕我手动删除了Z这个字母。
